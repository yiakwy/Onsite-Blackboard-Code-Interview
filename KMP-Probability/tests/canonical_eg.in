HERE IS A SIMPLE EXAMPLE
EXAMPLE
