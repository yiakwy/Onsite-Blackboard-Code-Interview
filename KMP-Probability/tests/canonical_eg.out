string to be matched:
 HERE IS A SIMPLE EXAMPLE
begin:
 EXAMPLE
       ^
       -
        EXAMPLE
              ^
              -
          EXAMPLE
                ^
            -
                EXAMPLE
                      ^
                      -
                  EXAMPLE
                        ^
                 -
